src/keystore.c

```diff
@@ -427,6 +427,20 @@ static bool _get_xprv(const uint32_t* keypath, const size_t keypath_len, struct
     return true;
 }
 
+bool get_root_fingerprint(char* fingerprint, int fingerprint_len)
```

Comment:
`keystore_get_rooot_fingerprint` -> it is good to prefix everything as in C, everything is in global namespace :(